app/assets/javascripts/activityChart.js

```diff
@@ -238,17 +238,28 @@
 
         // Update ARIA live region
         const liveRegion = document.getElementById('aria-live-account');
-            liveRegion.textContent = `Data updated for ${selectedText} - Last 7 Days`;
-        };
+        liveRegion.textContent = `Data updated for ${selectedText} - Last 7 Days`;
+
+        // Switch tables based on dropdown selection
+        const selectedTable = selectedValue === "individual" ? "table1" : "table2";
```

Comment:
I ran gulp and this is what I'm getting here:
![image](https://github.com/user-attachments/assets/96b6d24d-9816-46fe-a696-137675ac179a)

Is this PR addressing just this [ticket](https://github.com/GSA/notifications-admin/issues/1674)? or the toggle ticket as well?

Reply:
Spoke to Jonathan in slack and after merging with main, it seems to be working now